### PR TITLE
fix: explore organizations details

### PR DESCRIPTION
**Pull request by nishad-ayanworks**
## What ##
- fixed explore organizations details

### CODE REVIEW

File: src/components/organization/PublicOrganizationDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import { getPublicOrgDetails } from '../../api/organization'
+import { apiStatusCodes } from "../../config/CommonConstant";
+import ProfilesDesign from "../publicProfile/ProfilesDesign";
+import OrgWalletDetails from "../publicProfile/OrgWalletDetails";
+import OrgUserInfo from "../publicProfile/OrgUserInfo";
+import { AlertComponent } from "../AlertComponent";
+import CustomSpinner from "../CustomSpinner";
+
+
+interface OrgInterface {
```
Comment (tipusinghaw) [inline src/components/organization/PublicOrganizationDetails.tsx:12]: please add `interface` in separate file

File: src/components/organization/PublicOrganizationDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+        const { data } = response as AxiosResponse
+
+        if (data?.statusCode === apiStatusCodes?.API_STATUS_SUCCESS) {
+
+            setOrgData(data?.data);
+
+            const orgUsersFilterByRole = data?.data?.userOrgRoles?.filter(
+                (users: { orgRole: { name: string }; }) => {
+                    return users?.orgRole.name === "owner"
```
Comment (tipusinghaw) [inline src/components/organization/PublicOrganizationDetails.tsx:39]: I think we should use `ENUM` here

File: src/components/publicProfile/OrgUserInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
+import React from "react"
+import CustomAvatar from "../../components/Avatar"
+
+
+interface UserDetails {
```
Comment (tipusinghaw) [inline src/components/publicProfile/OrgUserInfo.tsx:5]: Please add `interface` in separate file

File: src/components/publicProfile/OrgWalletDetails.tsx [hunk 4]
```diff
@@ -0,0 +1,125 @@
+
+
+import React, { useEffect, useState } from "react"
+import { EmptyListMessage } from "../EmptyListComponent";
+import CustomQRCode from "../../commonComponents/QRcode";
+import DateTooltip from '../Tooltip';
+import { dateConversion } from '../../utils/DateConversion';
+
+
+interface OrgWalletDetailsObject {
```
Comment (tipusinghaw) [inline src/components/publicProfile/OrgWalletDetails.tsx:10]: Please add interface in separate file

File: src/components/publicProfile/ProfilesDesign.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
+import React from "react"
+import CustomAvatar from "../../components/Avatar"
+
+
+interface OrgInterface {
```
Comment (tipusinghaw) [inline src/components/publicProfile/ProfilesDesign.tsx:5]: Please add interface in separate file

File: src/components/organization/PublicOrganizationDetails.tsx (hunk 1 above)
Comment (nishad-ayanworks) [reply src/components/organization/PublicOrganizationDetails.tsx]: Resolved

File: src/components/publicProfile/OrgUserInfo.tsx (hunk 3 above)
Comment (nishad-ayanworks) [reply src/components/publicProfile/OrgUserInfo.tsx]: Resolved

File: src/components/publicProfile/OrgWalletDetails.tsx (hunk 4 above)
Comment (nishad-ayanworks) [reply src/components/publicProfile/OrgWalletDetails.tsx]: Resolved

File: src/components/organization/PublicOrganizationDetails.tsx (hunk 2 above)
Comment (nishad-ayanworks) [reply src/components/organization/PublicOrganizationDetails.tsx]: Resolved

**Review by tipusinghaw — APPROVED** [review]
LGTM

**Pull request merged**